src/global/constants.h

```diff
@@ -68,5 +72,8 @@ const int NOTSET = -1;
  */
 enum MessageSeverity { MSG_INFO = 1, MSG_ERROR };
 
+using CLOCK = std::chrono::system_clock;
+using TIMEPOINT = std::chrono::time_point<CLOCK>;
```

Comment:
Please, remove these from the header. If you think you need to share these definitions by the dependent classes (my understanding is that there only a few of those) then you may add a specialized header file that would have:
```
/// File: global/clock_defs.h

// System includes
#include <chrono>

using CLOCK = std::chrono::system_clock;
using TIMEPOINT = std::chrono::time_point<CLOCK>;
```